components/Domain/Bookmark/BookmarkSubHead/index.tsx

```diff
@@ -9,23 +9,27 @@ interface BookmarkSubHeadProps {
   editing: Boolean;
   setEditing: Dispatch<SetStateAction<Boolean>>;
   setAlertOpen: Dispatch<SetStateAction<Boolean>>;
+  count: number;
 }
 
 export default function BookmarSubHead({
   editing,
   setEditing,
   setAlertOpen,
+  count,
 }: BookmarkSubHeadProps) {
   return (
     <>
       <S.headLayout>
         <S.Frame>
-          {!editing && (
-            <>
-              <Body4 style={{ fontWeight: '500', color: '#BBBBBB' }}>4</Body4>
-              <Body4 style={{ color: '#BBBBBB' }}>개의 게시물</Body4>
-            </>
-          )}
+          <>
+            <Body4 state="emphasis" className="bookmarkSubHeadText">
+              {editing ? count : 4}
+            </Body4>
+            <Body4 state="emphasis" className="bookmarkSubHeadText">
+              {editing ? '개의 게시글이 선택됨' : '개의 게시글'}
+            </Body4>
```

Comment:
```suggestion
            <Body4 state="emphasis" className="bookmarkSubHeadText">
              {editing ? `${count}개의 게시글이 선택됨' : 4개의 게시글} 
            </Body4>
```
조건문이 겹치는거같은데 이렇게 합칠 수 있을거같아요!